src/app/components/LiveLabel/index.styles.ts

```diff
@@ -1,13 +1,65 @@
-import { css, Theme } from '@emotion/react';
+import { css, Theme, keyframes } from '@emotion/react';
+
+const pulseAnimation = keyframes({
+  '0%': {
+    transform: 'scale(1)',
```

Comment:
I believe we can get rid of these transforms (PS doesn't use it, and it doesn't seem to break it)